code/modules/shuttle/emergency.dm

```diff
@@ -249,17 +249,19 @@
 
 			if(time_left <= 50 && !sound_played) //4 seconds left - should sync up with the launch
 				sound_played = 1
+				var/hyperspace_sound = sound('sound/effects/hyperspace_begin.ogg')
 				for(var/area/shuttle/escape/E in world)
```

Comment:
The one acceptable use of `in world` searches.

Reply:
GLOB.areas?

Reply:
I recall this had problems associated with it, by doing this, because areas are kinda screwy in how they behave.